userspace/libsinsp/container_engine/cri.cpp

```diff
@@ -0,0 +1,176 @@
+/*
+Copyright (C) 2013-2018 Draios Inc dba Sysdig.
+
+This file is part of sysdig.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+#include "container_engine/cri.h"
+
+#include <sys/stat.h>
+#include <grpc++/grpc++.h>
+#include "cri.pb.h"
+#include "cri.grpc.pb.h"
+
+#include "container_engine/docker.h"
+#include "container_engine/mesos.h"
+#include <cri.h>
+#include "sinsp.h"
+#include "sinsp_int.h"
+
+using namespace libsinsp::cri;
+
+libsinsp::container_engine::cri::cri()
```

Comment:
I think with a `using namespace libsinsp;` you can avoid prefixing everything with `libsinsp`.  You might be able to do the same with `container_engine`.

Reply:
It worked except for mesos (we already have a class mesos in the top level namespace).